plugin/keyring_vault/tests/mtr/key_rotation_qa.test

```diff
@@ -27,6 +27,13 @@ call mtr.add_suppression("\\[ERROR\\] InnoDB: Failed to decrpt encryption inform
 
 --source generate_default_conf_files.inc
 
+# Create mount points
+--let MOUNT_POINT_SERVICE_OP=CREATE
```

Comment:
let FOO= vs let $BAR=
inconsistency on the dollar sign. By itself too minor to force a PR respin